src/main/java/pl/jaknauczycsieprogramowania/movielibrary/controllers/MovieController.java

```diff
@@ -0,0 +1,52 @@
+package pl.jaknauczycsieprogramowania.movielibrary.controllers;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+import pl.jaknauczycsieprogramowania.movielibrary.models.Movie;
```

Comment:
since this project is reflecting your code, would be good to have a package name reflecting it like (com.zaneta.movielibrary), the package will be more meaningful

Reply:
fixed